Cargo.toml

```diff
@@ -5,15 +5,19 @@ authors = [
   "The Servo project developers",
 ]
 description = "A binding generator for Rust"
-homepage = "https://github.com/servo/rust-bindgen"
 keywords = ["bindings", "ffi", "code-generation"]
+categories = ["external-ffi-bindings"]
```

Comment:
I thought this category was for bindings themselves, presumably not bindings generators. Would "develepment tools" make more sense?